### PR TITLE
Added GridspecLayout example

### PR DESCRIPTION
**Pull request by GeorgianaElena**
Added a new notebook that builds a multiplication table "game", using *GridspecLayout*.
It's my first time playing with widgets, so I'm not sure I did everything right :relaxed:

closes https://github.com/voila-dashboards/voila/issues/423

### CODE REVIEW

**Comment by jtpio** [comment]
Thanks @GeorgianaElena, this is a fun example!

Do you think we should point to the example notebooks (currently in the `notebooks` folder) somewhere in the [documentation](https://voila.readthedocs.io)?

**Comment by GeorgianaElena** [comment]
Thanks @jtpio, I really had fun working on it.

>Do you think we should point to the example notebooks (currently in the notebooks folder) somewhere in the documentation?

It seems that the example notebooks are mentioned a few times in the docs (e.g. [here](https://voila.readthedocs.io/en/stable/contribute.html#running-the-examples) and [here](https://voila.readthedocs.io/en/stable/deploy.html#setup-an-example-project)).
But I think it would be nice to have a dedicated section *"About the example notebooks"* (or some other (better) title), where we describe in a few words the notebooks and link to the section that explains [how to run them](https://voila.readthedocs.io/en/stable/contribute.html#running-the-examples).
I would put this in [*Deploying Voila*](https://voila.readthedocs.io/en/stable/deploy.html#deploying-voila), just before [set-upping an example project](https://voila.readthedocs.io/en/stable/deploy.html#setup-an-example-project) it's described.
What do you think?

**Comment by jtpio** [comment]
> But I think it would be nice to have a dedicated section _"About the example notebooks"_ (or some other (better) title), where we describe in a few words the notebooks and link to the section that explains [how to run them](https://voila.readthedocs.io/en/stable/contribute.html#running-the-examples).

That would be very valuable yes :+1: 

At the moment it's a bit hidden in the contributing guide, which is probably not the first thing end users look at.

**Comment by GeorgianaElena** [comment]
@jtpio, should I address this in the current PR or should I open a new one?

**Comment by jtpio** [comment]
@GeorgianaElena you're right this could be done in a different PR

File: notebooks/gridspecLayout.ipynb [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+    "# Which multiplication table do you want to learn?"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 8,
+   "metadata": {
+    "scrolled": false
+   },
+   "outputs": [
```
Comment (jtpio) [inline notebooks/gridspecLayout.ipynb]: I was quickly checking the other examples, and their outputs are removed from the notebook. So we should probably do the same here.

Comment (GeorgianaElena) [reply notebooks/gridspecLayout.ipynb]: Thanks @jtpio! I removed the cell outputs and added a notebook description. Will open a new PR for the docs.

**Pull request merged**